src/main/java/film/springbootapplication/controller/GenreController.java

```diff
@@ -4,31 +4,35 @@
 import film.springbootapplication.model.Genre;
 import film.springbootapplication.service.GenreService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import javax.persistence.EntityNotFoundException;
 import java.util.List;
 
 @RestController
-@RequestMapping( value = "/genres")
+@RequestMapping(value = "/genres")
 public class GenreController {
 
     @Autowired
-    GenreService genreService;
+    private GenreService genreService;
 
-    @GetMapping()
+    @GetMapping(value = "/genres")
```

Comment:
We already defined this mapping. Do we really need address like .../genres/genres ?